### PR TITLE
Making qtip buttons in calendar inline.

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

Fixing QTips in the calendar. The view edit buttons are not inline with the cross.

## Motivation and Context

* Regression issue

## How To Test This

* Compile Sass
* Hover over a calendar event
* The buttons in the qtip should be inline.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: custom/modules/Accounts/metadata/detailviewdefs.php [hunk 1]
```diff
@@ -0,0 +1,216 @@
+<?php
```
Comment (mattlorimer) [inline custom/modules/Accounts/metadata/detailviewdefs.php]: This shouldn't be here I assume

File: include/SugarEmailAddress/SugarEmailAddressLegacy.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/*********************************************************************************
```
Comment (mattlorimer) [inline include/SugarEmailAddress/SugarEmailAddressLegacy.js]: same here?

Comment (daniel-samson) [inline include/SugarEmailAddress/SugarEmailAddressLegacy.js]: Removed

File: custom/modules/Accounts/metadata/detailviewdefs.php (hunk 1 above)
Comment (daniel-samson) [inline custom/modules/Accounts/metadata/detailviewdefs.php]: Removed

**Comment by daniel-samson** [comment]
Files Removed

**Comment by samus-aran** [comment]
Styling is still not quite Right.
calendar

**Comment by daniel-samson** [comment]
I have made the spacing equidistant. The space is needed for the tablet users with big fingers.

**Pull request merged**